tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/PythonLanguageService.cs

```diff
@@ -137,9 +137,10 @@ public override async Task<PackageInfo> GetPackageInfo(string packagePath, Cance
 
     public override async Task<bool> RunAllTests(string packagePath, CancellationToken ct = default)
     {
-        var result = await processHelper.Run(new ProcessOptions(
```

Comment:
We can probably adopt the Options + Helper pattern used in the repo before for such helper methods.